compiler/rustc_codegen_ssa/src/back/link.rs

```diff
@@ -979,8 +979,9 @@ fn link_natively<'a>(
                      but `link.exe` was not found",
                 );
                 sess.note_without_error(
-                    "please ensure that VS 2013, VS 2015, VS 2017, VS 2019 or VS 2022 \
-                     was installed with the Visual C++ option",
+                    "please ensure that Visual Studio 2013 or later, or Build Tools \
+                     for Visual Studio were installed with the Visual C++ option. \
+                     Note: VS Code is a different product, and is not sufficient.",
```

Comment:
I like this wording better. Thanks for the clarification! As a small nitpick, I think the "Note: VS Code .." should be in a separate call to `sess.note_without_error` so that it formats the same as other notes do:

```suggestion
                    "please ensure that Visual Studio 2013 or later, or Build Tools \
                     for Visual Studio were installed with the Visual C++ option."
                 );
                 sess.note_without_error(
                     "VS Code is a different product, and is not sufficient."
                 );
```